### ISSUE TITLE
Unable to find services.yaml for the custom_updater integration in 0.92

### ISSUE THREAD

**Issue by kmlucy** [custom-components/custom_updater#131]
## Version of the custom_component
4.2.19

## Describe the bug
Get warning: ``Unable to find services.yaml for the custom_updater integration`` after updating to 0.92.

## `custom_updater` configuration

```yaml
custom_updater:
  track:
    - cards
    - components
  card_urls:
    - https://raw.githubusercontent.com/kalkih/mini-graph-card/master/tracker.json
    - https://raw.githubusercontent.com/kuuji/button-card/master/tracker.json
    - https://raw.githubusercontent.com/matisaul/vertical-style-card/master/tracker.json
    - https://raw.githubusercontent.com/kalkih/mini-media-player/master/tracker.json
    - https://raw.githubusercontent.com/kalkih/mini-graph-card/master/tracker.json
```

**Comment by callifo** [comment]
Its just missing the service.yaml file. Create a services.yaml file in the folder where custom_updater is, and add the following:

```
check_all:
  description: Check for custom_component updates
install:
  description: Install/Update specified custom_component
update_all:
  description: Update all custom_components
```

**Comment by kmlucy** [comment]
The ``custom_updater.py`` file was just in the root ``custom_components`` folder. When I created a ``custom_updater`` folder and moved ``custom_updater.py`` as well as ``services.yaml`` into it, I get this error:

```
Setup failed for custom_updater: Integration not found.
```

If I also add a blank ``__init__.py``, I get this error:

```
Error during setup of component custom_updater
Traceback (most recent call last):
  File "/usr/src/app/homeassistant/setup.py", line 156, in _async_setup_component
    component.setup, hass, processed_config)  # type: ignore
AttributeError: module 'custom_components.custom_updater' has no attribute 'setup'
```

**Comment by stepanov1975** [comment]
yep. adding  service.yaml to custom_components folder not working for me also

**Comment by callifo** [comment]
I didn't realise custom_updater supported its own folder as it doesn't match the naming scheme used by other custom components these days. I've always had it in the root of custom_components.

**Comment by DavidFW1960** [comment]
I have it in the root folder as well as per the instructions.

Kind of waiting for @ludeeus to confirm we should create our own services.yaml file as per the above or for him to update the component or otherwise comment on this issue. He dodged the question when I asked about this on discord when I got the warning in the beta for 0.92. It seems to work fine without it anyway. It’s just a warning, not an error.

**Comment by ludeeus** [comment]
And that is the issue, it's just a warning.
If I where to "fix" that I would break the component for everyone trying to update it ( the structure needs changing).
I'll leave it up to you (the user) if I should or not.

**Comment by DavidFW1960** [comment]
So then is it unwise to create a settings.yaml file as described above? I don’t understand how ‘fixing’ it would break the component for everyone trying to update it.... but I don’t understand a lot of things and that’s ok.....

**Comment by ludeeus** [comment]
<https://github.com/custom-components/custom_updater/issues/133>
@kmlucy @callifo @stepanov1975 @DavidFW1960 Have a look at that ☝️

**Comment by ludeeus** [comment]
As for not responding/dodging... well...
This started to be an "issue" in 0.92.0 this is what I have done since that release:
- Was sleeping.
- Woke up.
- Went to my day job (Still there).

I was aware of it during the beta, but due to server issues I have not looked into it myself.
Generally creating and publishing a fix for a custom_component during beta I would not have done (tried that before) most users do not run beta, and it could have messed up stuff for them.

As for this particular change the structure of the component needs to change to completely fix it, hence the vote in #133 .

**Comment by DavidFW1960** [comment]
Well I wasn’t trying to get you to fix it during the beta just asking for a response. When a component I use breaks in the beta I do make a point of reaching out to the dev for the component to see if they are aware and to ask for some guidance. I think that’s part of the point of the beta and most devs seem to be grateful for being made aware. Example for me as well as this component, my fire-service-danger broke and also the custom sun component both of which have been fixed now before it broke for everyone when 0.92 was released.

**Comment by james-fry** [comment]
> It seems to work fine without it anyway. It’s just a warning, not an error.

Today's WARNING is tomorrow's ERROR :)
Im sure this will be fixed before that time.
@ludeeus thanks for this component - its a lifesaver
